src/Corax/Indexing/IIndexEntryBuilder.cs

```diff
@@ -9,6 +9,7 @@ public interface IIndexEntryBuilder
     void Boost(float boost);
     ReadOnlySpan<byte> AnalyzeSingleTerm(int fieldId, ReadOnlySpan<byte> value);
     void WriteNull(int fieldId, string path);
+    void WriteNonExistingField(int fieldId, string path);
```

Comment:
Does it make sense to "write" it while it doesn't exist? Maybe "MarkNonExistingField"?

Reply:
I wanted to stay consistent with `WriteNull`. Even though `null` is an actual term both methods do pretty much the same, just writing to different posting list